towerlib/towerlib.py

```diff
@@ -1847,14 +1848,14 @@ def create_job_template(self,  # pylint: disable=too-many-arguments, too-many-lo
             raise InvalidProject(project)
         if playbook not in project_.playbooks:
             raise InvalidPlaybook(playbook)
+        if any([credential, credential_type]):
```

Comment:
This is wrong, it should be ```not all```. ```any``` would be true even if only one argument is set which would make the code always break for no reason.

Reply:
Implemented in the new commit